### DISCUSSION TITLE
API to get package downloads count per package version

### DISCUSSION THREAD

**Discussion by web-padawan** [npm/feedback#183] [Registry] [answered]
The npm registry has an old [API endpoint](https://github.com/npm/registry/blob/master/docs/download-counts.md) for getting downloads count per package.

Recently it was [announced](https://github.blog/changelog/2021-01-27-npmjs-com-displays-per-version-download-counts/) that npmjs.com now displays per-version download counts.

Are there any plans to provide another API endpoint to allow fetching download counts per version?

**Comment by web-padawan** [comment]
Please disregard this, found that the API is there but not yet documented.

**Comment by MylesBorins** [comment, marked as answer]
Here is an example API call for those who find this discussion

https://api.npmjs.org/versions/fastify/last-week
